substrafl/dependency/manage_dependencies.py

```diff
@@ -125,7 +131,13 @@ def local_lib_wheels(lib_modules: List[ModuleType], *, dest_dir: Path) -> List[s
                 cwd=str(lib_path),
                 text=True,
             )
-            wheel_name = re.findall(r"filename=(\S*)", ret)[0]
+            created_wheel_name = re.findall(r"filename=(\S*)", ret)[0]
+
+            if created_wheel_name != wheel_name:
+                raise exceptions.WrongWheelNameError(
+                    f"The `pip wheel` command built a wheel with an unexpected name: {created_wheel_name} "
+                    "instead of {wheel_name}."
```

Comment:
```suggestion
                   f "instead of {wheel_name}."
```
Not sure but the colour highlighting seems to indicate that you need to repeat the `f`

Reply:
you're right thanks!